parsers/IL.py

```diff
@@ -109,14 +106,23 @@ def fetch_production(
         raise NotImplementedError("This parser is not yet able to parse past dates")
 
     data = fetch_all()
-    production = [float(item) for item in data]
+    production = data.get("Production")
+    reserve = data.get("CurrentReserve")
+
+    if not production or not reserve:
+        raise ValueError("Could not parse Noga production")
+
+    production = production.replace(',', '')
+    reserve = reserve.replace(',', '')
+
+    production = float(production) * 1000 + float(reserve) * 1000
 
     # all mapped to unknown as there is no available breakdown
     return {
         "zoneKey": zone_key,
         "datetime": arrow.now(TZ).datetime,
```

Comment:
NOGA has the timestamp on their web site (e.g.  Last update: 23/05/2023 19:04:43 ); it would be great if the parser could use that instead of the current timestamp